scripts/python3.8/linux/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# This scripts builds a Cura AppImage using a docker container. The docker base
+# image it uses is "cura-build-env:centos7", which can be created with the
+# cura-build-environment repository. The AppImage will be created under the
+# directory "appimages", which will be created if it's not present. Note that
+# the AppImage being created will not be signed.
+#
+
+set -e
+
+# Get where this script resides
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../../.."
+
+# Cura release configurations
+CURA_BUILD_ENV_DOCKER_IMAGE="${CURA_BUILD_ENV_DOCKER_IMAGE:-ultimaker/cura-build-environment:centos7-python3.8-qt5.15.2}"
```

Comment:
Should say `cura-build-environment:master`, right?

Reply:
We can put it as master there, yes. We need to create one environment with that tag though :D